packages/kyt-cli/cli/actions/setup.js

```diff
@@ -336,29 +405,48 @@ module.exports = (config, flags, args) => {
       createSrcDirectory();
       createGitignore();
       copyStarterKytFiles();
-      removeTmpDir();
+      removeTmpRepo();
       logger.end(`Done adding starter kyt: ${repoURL}`);
     };
 
     // First, clean any old cloned repositories.
-    removeTmpDir();
-    simpleGit.clone(repoURL, tmpDir, {}, afterClone);
+    removeTmpRepo();
+    simpleGit.clone(repoURL, tmpRepo, {}, afterClone);
   };
 
-  // default setup flow
-  const defaultSetup = () => {
-    logger.start('Setting up kyt');
-    updateUserPackageJSON(true);
-    createEditorconfigLink();
-    createESLintFile();
-    createStylelintFile();
-    createKytConfig();
-    createGitignore();
-    logger.end('Done setting up kyt');
+  const starterKytPrompt = () => {
+    const question = [
+      {
+        type: 'list',
+        name: 'starterChoice',
+        message: 'Which starter-kyt would you like to install?', // eslint-disable-line
+        choices: ['Universal', 'Static'],
```

Comment:
probably ok for now but it would be cool to have this assembled by just reading the contents of `packages/starter-kyts`, or at least some configuration somewhere?

also, maybe it's worth having a third option for "Specify a remote repository" where you don't necessarily need knowledge of the `-r` flag?